.github/workflows/docker-build-push-production.yml

```
@@ -0,0 +1,90 @@
+name: Docker build and push
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  AWS_REGION: ca-central-1
+  FUNCTION_PREFIX: gcds-api 
+  GITHUB_SHA: ${{ github.sha }}
+  REGISTRY: 307395567143.dkr.ecr.ca-central-1.amazonaws.com/gc-design-system/api 
+
+permissions:
+  id-token: write
+  contents: write
+  pull-requests: write
+  actions: write
+  checks: write
+  statuses: write
+  security-events: write
```

I think this can be adjusted to just the following (the example I gave you is wrong 😅)
```suggestion
permissions:
  id-token: write
  contents: write
  security-events: write
```